### PR TITLE
Docs update for v0.2.0 release

### LINKED ISSUE #472: Build Release v0.2.0
This issue to create a release once the milestone `v0.2.0` is achieved,

### PR DESCRIPTION
**Pull request by anusha94**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #472

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -42,33 +42,6 @@ Note for macOS users: you may need to [increase the memory available](https://do
 
 ## Initialize the management cluster and install the BringYourOwnHost provider
 
-To initialize Cluster API Provider BringYourOwnHost, `clusterctl` requires the following settings, which should
```
Comment (sachinkumarsingh092) [inline docs/getting_started.md:45]: Does clusterctl now has BYOH as a default provider?

Comment (anusha94) [reply docs/getting_started.md:45]: Not as a default, but you don't have to explicitly add it to `clusterctl.yaml`

The changes were made in this PR - https://github.com/kubernetes-sigs/cluster-api/pull/5593 and is part of CAPI `v1.1`

**Review by sachinkumarsingh092 — APPROVED** [review]
lgtm

**Review by dharmjit — APPROVED** [review]
lgtm

**Pull request merged**